src/Tribe/Attendee_Registration/View.php

```diff
@@ -300,47 +88,13 @@ public function get_cart_provider( $provider ) {
 	 * @return string The class string or empty string if provider not found or not active.
 	 */
 	public function get_form_class( $provider ) {
-		$class = '';
-
-		if ( is_string( $provider ) ) {
-			$provider = Tribe__Tickets__Tickets::get_ticket_provider_instance( $provider );
-		}
-
-		if ( ! empty( $provider ) ) {
-			$provider = $provider->attendee_object;
-		}
-
-		if ( empty( $provider ) ) {
-			/**
-			 * Allows filtering the class before returning it in the case of no provider.
-			 *
-			 * @since 4.10.4
-			 *
-			 * @param string $class The (empty) class string.
-			 */
-			return apply_filters( 'tribe_attendee_registration_form_no_provider_class', $class );
-		}
-
-		/**
-		 * Allow providers to include their own strings/suffixes.
-		 *
-		 * @since 4.10.4
-		 *
-		 * @param array $provider_classes In the format of: $provider -> class suffix.
-		 */
-		$provider_classes = apply_filters( 'tribe_attendee_registration_form_classes', [] );
+		try {
+			/* @var Tribe__Tickets_Plus__Attendee_Registration__View $view */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: a309b84c32349e10ac98ec300d6fd672</sub>